test/test_rosdep_debian.py

```diff
@@ -64,9 +64,9 @@ def test_read_apt_cache_showpkg():
     m = Mock()
     with open(os.path.join(get_test_dir(), 'showpkg-curl-wget-libcurl-dev'), 'r') as f:
         m.return_value = f.read()
-
-    results = list(_read_apt_cache_showpkg(['curl', 'wget', '_not_existing', 'libcurl-dev'], exec_fn=m))
-    assert len(results) == 4, results
+    pkgs = ['curl', 'wget', '_not_existing', 'libcurl-dev', 'ros-kinetic-rc-genicam-api']
```

Comment:
will this still work as intended after the next kinetic sync, from which on ros-kinetic-rc-genicam-api will be available?

Reply:
Yes, this just tests the format parser based on the text file.

Reply:
ah, right...